src/server/api/controllers/eventDetail.controller.js

```diff
@@ -0,0 +1,25 @@
+const knex = require('../../config/db');
+
+const eventDetail = async (eventId) => {
+  const details = await knex('events')
+    .select(
+      'users.name as Student name',
+      'questions.description',
+      'events.event_date as Event date',
+      'topics.topic_name as Topic',
+      'topics.week_number as Week number',
+      'topics.homework_url as Link',
+      'organizations.name as Organization name',
```

Comment:
```suggestion
      'users.name as name',
      'questions.description',
      'events.event_date as eventDate',
      'topics.topic_name as topic',
      'topics.week_number as weekNumber',
      'topics.homework_url as link',
      'organizations.name as organization',
```
This is `camelCase`. JSON properties usually do not have spaces in them.